doc/ledger3.texi

```diff
@@ -6697,6 +6697,57 @@ $ ledger reg Expenses --group-by "payee" --group-title-format "-----------------
 ...
 @end smallexample
 
+@item --hashes @var{ALGO}
+Records the chained hash of each transaction in a @var{Hash} metadata
+value, according to the hashing algorithm given by the @var{ALGO}
+argument (at the moment, only @code{sha512} is supported). To use this,
+record the @var{Hash} metadata explicitly in some of your transactions;
+these will be checked against the hashes calculated internally, and if
+they do not match, an error is reported. You may also write just a
+prefix of the @var{Hash}, which is less verbose but still gives quite
+good assurance.
+
+The support algorithms are:
+
+@table @code
+@item sha512
+Use the SHA512 hashing algorithm.
+@item sha512_256
```

Comment:
Is `sha512_256` a common and known identifier for the first 256 bits of a SHA512 hash? If yes, what other software makes use of these? I'm curious.

Reply:
We use it for the blockchain at my place of work, and I've seen it as an algorithm in other crypto libraries. If you Google around, you'll see that it's a pretty well-known thing.